### ISSUE TITLE
NaN loss on cotatron_trainer

### ISSUE THREAD

**Issue by manhlab** [mindslab-ai/assem-vc#21]
I got train loss=NaN when trained cotatron_trainer. 
Can you upload pretrained checkpoint or how to fix this?

**Comment by wookladin** [comment]
Hi. Which data did you use?

We uploaded the pre-trained weight of Cotatron and corresponding configuration in [this link](https://drive.google.com/drive/folders/1aIl8ObHxsmsFLXBz-y05jMBN4LrpQejm?usp=sharing).
You can start transfer learning from that checkpoint.

Please check out!

**Comment by wookladin** [comment]
No updates, so I'm closing this issue.
Please open the issue again if you need.

**Comment by blx0102** [comment]
@manhlab You could try to check your tools' version, I got train loss=NaN as well when torch=1.7, but with torch=1.4, the loss is reasonable.
